terraform/vars/terraform-production.tfvars

```diff
@@ -5,6 +5,8 @@ rw_api_url                = "https://api.resourcewatch.org"
 desired_count             = 2
 auto_scaling_min_capacity = 2
 auto_scaling_max_capacity = 15
+fargate_cpu               = 2048
+fargate_memory            = 4096
```

Comment:
I guess this is OK, but somehow you are including hotfixes from master into your change?